content/api/users/index.md

```diff
@@ -86,8 +86,8 @@ Upload a new avatar
 
 | Parameter  | Type                      | Description                |
 | ---------- | ------------------------- | -------------------------- |
-| __`User`__ | [`smart id`](../smart-id) | Required. The user to edit |
-| `Picture`  | `string`                  | Uploaded file              |
+| __`UserID`__ | [`smart id`](../smart-id) | Required. The user to edit |
```

Comment:
Since this is a "smart ID" field, this should remain as "User". With smart IDs, you can use "User.ID" and specify the user's numeric ID or "User.Name" and specify the user's username.